tests/tools/test_pandas.py

```diff
@@ -19,24 +19,19 @@
     ),
     (
         {"f": [[0], [0, 1]], "ff": ["k", None], "f_0": [2, None]},
-        {
-            "f_0": [2.0, None],
-            "f_1": [None, 1],
-            "ff": ["k", None],
-            "f_5ecd5": [0.0, 0.0],
-        },
-        {"f_1": "f", "f_5ecd5": "f"},
+        {"f_0": [2.0, None], "f_1": [None, 1], "ff": ["k", None], "f_0_5ecd5": [0, 0]},
+        {"f_1": "f", "f_0_5ecd5": "f_0"},
```

Comment:
We need mapping to the original column, in this case `f_0_5ecd5` comes from `f` originally.